docs/DeveloperGuide.md

```diff
@@ -4,10 +4,154 @@
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
-## Design & implementation
+## Design
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Parser
+API: [Parser.java]({repoURL}src/main/java/seedu/cafectrl/parser/Parser.java)
 
+![Parser Class Diagram](images/class/Parser.png)
```

Comment:
The accessibility of a variable or method should be + (public) or - (private) instead of colours by default in plantUML